NYCJobsWeb/JobsSearch.cs

```diff
@@ -39,44 +39,44 @@ static JobsSearch()
             }
         }
 
-        public DocumentSearchResult<Document> Search(string searchText, string businessTitleFacet, string postingTypeFacet, string salaryRangeFacet,
+        public SearchResults<SearchDocument> Search(string searchText, string businessTitleFacet, string postingTypeFacet, string salaryRangeFacet,
             string sortType, double lat, double lon, int currentPage, int maxDistance, string maxDistanceLat, string maxDistanceLon)
         {
             // Execute search based on query string
             try
             {
-                SearchParameters sp = new SearchParameters()
+                SearchOptions sp = new SearchOptions()
                 {
                     SearchMode = SearchMode.Any,
-                    Top = 10,
+                    Size = 10,
                     Skip = currentPage - 1,
                     // Limit results
-                    Select = new List<String>() {"id", "agency", "posting_type", "num_of_positions", "business_title", 
-                        "salary_range_from", "salary_range_to", "salary_frequency", "work_location", "job_description",
-                        "posting_date", "geo_location", "tags"},
+                    //Select = new List<String>() {"id", "agency", "posting_type", "num_of_positions", "business_title", 
```

Comment:
Why commented out?  Should we remove this code?

Reply:
Yes, I have removed it.